src/xc_integrator/local_work_driver/device/common/uvvars.hpp

```diff
@@ -7,15 +7,21 @@
  */
 #pragma once
 #include "device/xc_device_task.hpp"
+#include "device/xc_device_data.hpp"
 #include "device/device_queue.hpp"
 
 namespace GauXC {
 
 void eval_uvvars_lda( size_t ntasks, int32_t nbe_max, int32_t npts_max,
   XCDeviceTask* device_tasks, device_queue queue );
 
+void eval_uvvars_lda_uks( size_t ntasks, int32_t nbe_max, int32_t npts_max,
```

Comment:
Much like the CPU code, I'd like to have the `rks` functions labeled as well as a part of this PR